### DISCUSSION TITLE
Управление документами в проектной организации

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#118] [Сделанные проекты]
Доброго времени суток!
В нашей организации работает система  ведения документов и документооборота.
До появления Totum пользовались Bitrix, она оказалась не столь гибкой как Totum.
Сейчас мы преобразовали наш проект:
![image](https://user-images.githubusercontent.com/103171127/164715098-f1b2cd7e-d362-4ad9-90c7-019d1c319a58.png)
![image](https://user-images.githubusercontent.com/103171127/164715738-504b51a9-a479-46ff-acca-b5c06d8d3594.png)

Огромную роль сыграла документация и учебные курсы на русском языке, и форма лицензирования.

**Comment by mihail-cornev** [comment]
Вижу раздел: Письма. Как вы наладили работу с почтой? Или это другое что-то?
